go.mod

```diff
@@ -1,64 +1,93 @@
 module github.com/redhat-developer/observability-operator/v3
 
-go 1.16
+go 1.19
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v1.2.3
 	github.com/goccy/go-yaml v1.9.5
-	github.com/integr8ly/grafana-operator/v3 v3.10.3
+	github.com/grafana-operator/grafana-operator/v4 v4.7.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.17.0
+	github.com/onsi/gomega v1.20.1
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/operator-framework/api v0.3.20
-	github.com/operator-framework/operator-registry v1.12.6-0.20200611222234-275301b779f8
+	github.com/openshift/cluster-logging-operator v0.0.0-20221118094835-2d8faae5596f
+	github.com/operator-framework/api v0.17.1
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-operator/prometheus-operator v0.43.0
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.55.1
+	github.com/prometheus-operator/prometheus-operator v0.58.0
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.58.0
 	github.com/prometheus/client_golang v1.13.0
 	github.com/sirupsen/logrus v1.8.1
-	k8s.io/api v0.23.5
-	k8s.io/apimachinery v0.23.5
+	k8s.io/api v0.24.3
+	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
Hey, looks good to me but I would remove the replaces `replace k8s.io/client-go => k8s.io/client-go v0.24.3` and update it directly in require

Reply:
Thanks @MStokluska. I had to revert back to using replace here after rebasing after another PR. I'll check again if we can avoid the replace

Reply:
@MStokluska Don't think we can get rid of the replace here. We need to support install of [cluster-logging-operator](https://github.com/openshift/cluster-logging-operator) which is using `k8s.io/client-go v12.0.0+incompatible`